### PR TITLE
Manage quizzes feature

### PR DESCRIPTION
**Pull request by yuvrajsab**

### CODE REVIEW

File: app/Http/Controllers/Admin/QuizController.php [hunk 1]
```diff
@@ -66,4 +68,77 @@ public function evaluate(Quiz $quiz)
...
+
+        flash('Quiz updated!')->success();
+
+        return redirect()->route('admin.quizzes.index');
+    }
+
+    public function delete(Quiz $quiz)
+    {
+        // can only delete quiz which is not active
+        abort_if($quiz->isActive, 403, "You can only delete quiz which is'nt active");
```
Comment (rumansaleem) [inline app/Http/Controllers/Admin/QuizController.php]: Typo here: ~~is'nt~~ "isn't"

File: resources/views/admin/quizzes/create.blade.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    </div>
+    <div class="card-content">
+        <form action="{{ route('admin.quizzes.store') }}" method="POST">
+            @csrf
+
+            <label class="control">
+                Title
+            </label>
+            <input class="control" name="title" type="text" placeholder="Sankalan" value="{{ old('title') }}" required>
+
```
Comment (rumansaleem) [inline resources/views/admin/quizzes/create.blade.php]: Group every label input pair inside a div with bottom margin

File: resources/views/admin/quizzes/index.blade.php [hunk 3]
```diff
@@ -73,6 +76,13 @@ class="btn is-green is-sm font-normal"
                                 class="btn is-blue is-sm font-normal">
                                 Evaluate
                             </button>
+                            <br>
+                            <form v-if="!quiz.isActive" class="inline-block" action="{!! route('admin.quizzes.delete', $quiz) !!}" method="POST">
```
Comment (rumansaleem) [inline resources/views/admin/quizzes/index.blade.php]: Why are you using  `{!! !!}` ?

**Review by rumansaleem — APPROVED** [review]

File: resources/views/admin/quizzes/index.blade.php (hunk 3 above)
Comment (yuvrajsab) [reply resources/views/admin/quizzes/index.blade.php]: ..It must be a mistake

**Pull request merged**